app/src/main/java/hr/foi/air/servicesync/navigation/AppNavHost.kt

```diff
@@ -82,4 +82,30 @@ fun NavGraphBuilder.AppNavHost(navController: NavHostController) {
             }
         )
     }
+    composable("addReview/{companyId}/{userId}") { backStackEntry ->
+        val context = LocalContext.current
+        val companyId = backStackEntry.arguments?.getString("companyId") ?: ""
+        val userId = backStackEntry.arguments?.getString("userId") ?: ""
+
+        AddReviewScreen(
+            navController = navController,
+            companyId = companyId,
+            userId = userId,
+            onReviewSubmit = { success ->
+                if (success) {
+                    Toast.makeText(
+                        context,
+                        "Review added successfully.",
+                        Toast.LENGTH_SHORT
+                    ).show()
+                } else {
+                    Toast.makeText(
+                        context,
+                        "Failed to add review. Please try again.",
+                        Toast.LENGTH_SHORT
+                    ).show()
```

Comment:
I don't see why appnavhost should know what to do when a review is submitted or failed.

Reply:
I added it just for info when review is submitted.